stanford-sw/test/data/closedLibraries.xml

```diff
@@ -0,0 +1,122 @@
+<records xmlns='http://www.loc.gov/MARC21/slim'>
```

Comment:
in a perfect world, I prefer record creation inline in the test so you can see what data you are testing in one shot.  Not worth redoing here.  Just something to keep in mind for future new test data.  

If there had been time, I would have gotten rid ofall the marc data test files and put it all inline.  :-P